Fix pb inputs validation (#2958)
demisto/demisto-sdk: CHANGELOG.md

* Fixed an issue where **download** command failed when running on a beta integration.
* Added the `DEMISTO_SDK_GRAPH_FORCE_CREATE` environment variable. Use it to force the SDK to recreate the graph, rather than update it.
* Added support for code importing multi-level ApiModules to **lint**.

> * Fixed a bug where **validate** returned error on playbook inputs.

Reviewer: ```suggestion
* Fixed an issue where **validate** returned error on playbook inputs.
```

what inputs? in which cases?
